cmake/compilers/MSVC.cmake

```diff
@@ -30,6 +30,10 @@ if (MSVC_VERSION LESS_EQUAL 1900)
     # https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-1-c4503
     set(TBB_TEST_COMPILE_FLAGS ${TBB_TEST_COMPILE_FLAGS} /wd4503)
 endif()
+# Warning suppression C4127: conditional expression is constant
+set(TBB_TEST_COMPILE_FLAGS ${TBB_TEST_COMPILE_FLAGS} /wd4127)
+# Warning suppression C4800: forcing value to bool 'true' or 'false' (performance warning)
+set(TBB_TEST_COMPILE_FLAGS ${TBB_TEST_COMPILE_FLAGS} /wd4800)
```

Comment:
Do we really want to suppress this warnings unconditionally? We will not test that public header files do not produce these warnings.

Reply:
Yes, you're right, the local suppression seems like a better solution for this.